packages/components/src/Popover/Popover.css

```diff
@@ -56,6 +59,10 @@
   clip-path: polygon(-4px -4px, 15.3px 0, 0 15.3px);
 }
 
+:global(.jobber-retheme) .arrow {
```

Comment:
@chris-at-jobber should popover keep the arrow like the SG-1 version? I'm not sure on it's usage and if the arrow is still necessary.

Reply:
Hmm maybe

It is usually attached to an element, but so is Menu/Dropdown and we aren't using it there

I can see a case for leaving it